fmf/base.py

```diff
@@ -24,6 +24,7 @@
 SUFFIX = ".fmf"
 MAIN = "main" + SUFFIX
 IGNORED_DIRECTORIES = ['/dev', '/proc', '/sys']
+ADJUST_CONTROL_KEYS = ['because', 'continue', 'when']
```

Comment:
Perhaps this should be a set instead of a list?

Reply:
Could be, but `list` is ok as well, I'd say. Let's not block on this.